base/src/main/groovy/org/asciidoctor/gradle/base/AbstractAsciidoctorBaseTask.groovy

```diff
@@ -638,6 +635,11 @@ abstract class AbstractAsciidoctorBaseTask extends DefaultTask {
         this.outDir = createDirectoryProperty(project)
     }
 
+    @Nested
```

Comment:
@lptr Why is this change necessary for Gradle 7?

Reply:
It is not allowed to annotate fields anymore. This is to replace the annotation on the `configuredOutputOptions` field on line 89.

Reply:
Ah. Noted.